pycontrails/models/humidity_scaling.py

```diff
@@ -568,20 +569,27 @@ def scale(
 
 
 @functools.cache
-def _load_iagos_quantiles() -> npt.NDArray[np.float64]:
-    path = pathlib.Path(__file__).parent / "quantiles" / "iagos_quantiles.npy"
-    # FIXME: Recompute to avoid the divide by 100.0 here
-    return np.load(path, allow_pickle=False) / 100.0
+def _load_quantiles() -> pd.DataFrame:
+    """Load precomputed ERA5 and IAGOS quantiles.
 
-
-@functools.cache
-def _load_era5_ensemble_quantiles() -> npt.NDArray[np.float64]:
-    path = pathlib.Path(__file__).parent / "quantiles" / "era5_ensemble_quantiles.npy"
```

Comment:
I think you should create a directory `models/humidity_scaling/` and then include this file in there along with a sub module `humidity_scaling`. 

You can add an `__init__` to expose the methods/classes you need in there.

I'd like to keep static data alongside the models

Reply:
1c61242657a9ccab0245473d6feaa4816e5fd7a2